src/av2/geometry/utm.py

```diff
@@ -0,0 +1,105 @@
+# <Copyright 2022, Argo AI, LLC. Released under the MIT license.>
+
+"""Utilities for converting AV2 city coordinates to UTM or WGS84 coordinate systems."""
```

Comment:
Could we add some information about these two coordinate systems here?

Reply:
Sure, I added some references.